modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormAdminDisplayContext.java

```diff
@@ -1240,6 +1240,10 @@ protected DDMForm createSettingsDDMForm(
 			DDMFormField ddmFormField = ddmFormFieldsMap.get("storageType");
 
 			ddmFormField.setReadOnly(true);
+
+			ddmFormField = ddmFormFieldsMap.get("objectDefinitionId");
```

Comment:
@mateussgn can you sort and put objectDefinitionId first? Thanks 😄

Reply:
Sure 😃